client/state/happychat/constants.js

```diff
@@ -1,2 +1,7 @@
+export const HAPPYCHAT_CONNECTION_ERROR_FORCED_CLOSE = 'forced close';
+export const HAPPYCHAT_CONNECTION_ERROR_PING_TIMEOUT = 'ping timeout';
+export const HAPPYCHAT_CONNECTION_ERROR_TRANSPORT_CLOSE = 'transport close';
+export const HAPPYCHAT_CONNECTION_ERROR_TRANSPORT_ERROR = 'transport error';
+
```

Comment:
What are these constants for? Apart from `HAPPYCHAT_CONNECTION_ERROR_PING_TIMEOUT` I haven't seen them in use anywhere.